src/data_generators/overlap_data_set.py

```diff
@@ -30,11 +30,15 @@
 
 
 class OverlapDataSet(DataSet):
-    """Construct a multi-pub DataSet from a list of single-pub PublisherData.
+    """A DataSet constructed from a list of PublisherData by overlapping their reached sets.
 
     Once we have a list of PublisherData for each publisher, this class
     generates the cross-publisher reach overlap, relabels the reached ids to
     reflect the overlap, and finally includes these labeled ids to a DataSet.
+
+    In other words, the universes are overlapped while for each user, their
```

Comment:
The wording here is not clear to me. I didn't understand what it means! Do you mean that condition on having a fixed overlap for universes as input, then the frequency distribution is generated?